integration_tests/dbt_project.yml

```diff
@@ -64,8 +64,11 @@ vars:
     quickbooks_vendor_identifier: "vendor_data"
 
 
-  # For validation testing. This exclusion is only needed for validation testing up to the v0.14.0 release. It can be leveraged if needed in future package updates.
-  # account_type_exclusions: ('Discount')
+  # For validation testing.  
+  # This below vars configuration is needed for the `general_ledger_amounts_match` macro.
+  using_bill: true
+  using_credit_card_payment_txn: true
```

Comment:
Since the default for `using_credit_card_payment_txn` is false but we're setting it true here, should we set it to false in [this line](https://github.com/fivetran/dbt_quickbooks/blob/feature/multicurrency-support/.buildkite/scripts/run_models.sh#L22) so one run tests true and the other false? Or, you could just comment out this L70 when sending to buildkite?

Also I'm not sure if it would be related to this PR, it looks like the default value is set as false everywhere except for [here](https://github.com/fivetran/dbt_quickbooks/blob/main/macros/get_enabled_unioned_models.sql#L27). Is that intentional?

Reply:
@fivetran-catfritz Oh good catch. This should have been commented out. 

In the second link, I believe this is false by default? It's conditional on the variable being true.

Reply:
@fivetran-avinash According to [the docs](https://docs.getdbt.com/reference/dbt-jinja-functions/var#variable-default-values), `{% var('using_credit_card_payment_txn', True) %}` means the default value is True.
The line `{% if var('using_credit_card_payment_txn', True) %}` simply evaluates if the variable value provided is true/false.

Reply:
@fivetran-catfritz Ahh, good catch! Yes it should probably be consistent then.

Interestingly, I don't think the behavior changes very much unless the variable is not set, but since it is set by default the cases that it wouldn't be set are probably rare, which is why this hasn't been flagged.